Makefile

```diff
@@ -3,8 +3,15 @@ CC = cc
 
 PREFIX = /usr/local
 
-INCS   = -I/usr/local/include
-LINKS  = -L/usr/local/lib -lX11
+OS != uname -s
+.if ${OS} == OpenBSD
+INCS  = -I/usr/X11R6/include
+LINKS = -L/usr/X11R6/lib -lX11
+.else
+INCS  = -I/usr/local/include
+LINKS = -L/usr/local/lib -lX11
+.endif
+
```

Comment:
How about something like:
```
XPREFIX = /usr/local
OS != uname
.if ${OS} == OpenBSD
XPREFIX = /usr/X11R6
.endif
```
Alter INCS and LINKS accordingly. Should look cleaner and easier to read.